### PR TITLE
Remove iPhone Configuration Utility

### PR DESCRIPTION
**Pull request by Amorymeltzer**
All the links are dead; it no longer properly exists.  It has been [replaced](https://en.wikipedia.org/wiki/Apple_Configurator) Apple with the [Apple Configurator](https://www.apple.com/support/business-education/apple-configurator/) which appears to be Yosemite- and MAS-only.

Apparently I've got it on my machine (who knew?) and even the help link from the application itself is dead.

### CODE REVIEW

**Comment by radeksimko** [comment]
Based on some Googling, Apple seems to be replacing this tool with another one having different name: https://forums.plex.tv/index.php/topic/131873-iphone-configuration-utility-for-mac-os-x/?p=792500

I'm not sure if there's any available URL to dmg or any other distributable format. Even if there is, it most likely won't be mentioned in any Apple/official website as they always strive to push MAS as the main distribution channel.

https://itunes.apple.com/us/app/apple-configurator/id434433123?mt=12

To sum up, it may be worth _grepping_ throughout the downloaded app and eventually adding it to [`homebrew-unofficial`](https://github.com/caskroom/homebrew-unofficial), but agreed, it should disappear from this main repo.
